session-1-jigsaw-intro/07_patch_module_option/src/java.base/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -6,10 +6,24 @@
 import java.util.Set;
 
 public class ConcurrentHashMap<K,V> extends AbstractMap<K,V> implements ConcurrentMap<K,V>, Serializable {
+
 	public static void main(String[] args) {
 		System.out.printf("ConcurrentHashMap");
 	}
 
+	public ConcurrentHashMap() {
+	}
+
+	public ConcurrentHashMap(Map<? extends K, ? extends V> m) {
+	}
```

Comment:
Getting below when I make your changes:
Error occurred during initialization of VM
java.lang.IllegalAccessError: tried to access method java.util.concurrent.ConcurrentHashMap.<init>()V from class java.lang.WeakPairMap
        at java.lang.WeakPairMap.<init>(java.base/WeakPairMap.java:47)
        at java.lang.Module.<clinit>(java.base/Module.java:266)